docs/guides/authjs-migration.mdx

```diff
@@ -0,0 +1,373 @@
+---
+title: Migrating to Clerk from Auth.js
+description: Migrating from Auth.js
+---
+
+<TutorialHero
+  framework="authjs"
+  exampleRepoTitle="Migration Script Repository"
+  exampleRepo={[ 
+    {
+      title: "Migration Script",
+      link: "https://github.com/clerk/migration-script"
+      
+    }
+  ]}
+
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+    }
+  ]}
+>
+
+- Install `@clerk/nextjs`
+- Set environment keys
+- Wrap your Next.js app in `<ClerkProvider />`
+- Setup Sign-up and Sign-in UI
+- Protect your app
+- Read user and session data
+- User IDs as Foreign Keys
+- Create a production instance
+- Migrate users to a production instance
+- Further support
+
+</TutorialHero>
+
+This guide shows how to migrate an application using Auth.js (formerly NextAuth.js) to use Clerk for authentication.
```

Comment:
We need an H2 here, it can either be something like "Introduction" or "Overview" :)
I use "Introduction" on other pages but what ideas do you have for a heading here?

```suggestion
## Introduction

This guide shows how to migrate an application using Auth.js (formerly NextAuth.js) to use Clerk for authentication.
```